bin/ncp/CONFIG/nc-init.sh

```diff
@@ -152,7 +152,7 @@ EOF
   ncc app:disable updatenotification
 
   # News dropped support for 32-bit -> https://github.com/nextcloud/news/issues/1423
-  if ! [[ "$(uname -m)" =~ "armv7" ]]; then
+  if ! [[ "$(dpkg --print-architecture)" = "armhf" ]]; then
```

Comment:
```suggestion
  if [[ "$(dpkg --print-architecture)" != "armhf" ]] || [[ ! "$(uname -m)" =~ "armv7" ]]; then
```
This one should have the exclamation mark inside the double-brackets as we're doing a string comparison, we can also add the other method here with an OR (||) statement and negate the regex (=~) of the 2nd if-statement inside the bracket ( [[ ! ), like I suggested, or outside ( ! [[ ).